File: modul 5/PRAK504-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
int reverse(int a){
    int rev = 0;
    while(a !=0){
        rev = rev * 10 + a % 10;
        a /= 10;}
    return rev;}
int main(){
    int A,B;
    scanf("%d %d",&A, &B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf ("%d", reverse(C));
}

File: modul 5/PRAK504-2410817320004-AmandhaCitraMustika.py
def reverse(a) :
    rev = 0
    while a != 0 :
        rev = rev * 10 + a % 10
        a = a // 10
    return rev

A,B=map(int,input().split())
A=reverse(A)
B=reverse(B)
C=A+B
print(f"{reverse(C)}")